### PR TITLE
feat: navigation and modal added

### PR DESCRIPTION
**Pull request by mayuresh-tech**

### CODE REVIEW

File: components/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+      .nav-box-bar #nav-section-mobile-bar,
+      .nav-box-bar #mobile-section-bar {
+        display: none;
+      }
+      
+      .nav-box-bar #logo-bar {
+        text-align: center;
+      }
+      
+      #logo-bar a {
```
Comment (deekshithmd) [inline components/navigation/navigation.css:30]: Suggestion: try to avoid using id if possible

File: components/navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+      .nav-box-bar #nav-section-bar {
+        display: none;
+      }
+      
+      .nav-box-bar #nav-section-mobile-bar {
+        flex-grow: 1;
+      }
+      
+      #burger-bar div {
+        height: 0.125rem;
```
Comment (deekshithmd) [inline components/navigation/navigation.css]: for size less than 1rem try to use px

**Review by deekshithmd — APPROVED** [review]

File: components/navigation/navigation.css (hunk 2 above)
Comment (mayuresh-tech) [reply components/navigation/navigation.css]: Noted

**Comment by shruthipatil11** [comment]
```
   .nav-box-bar #nav-section-mobile-bar,
      .nav-box-bar #mobile-section-bar {
        display: none;
      }
```
Try to use classes as classes can be reused, in order to more specific , you can increases the specificity ..

Overall look and feel of UI and code looks good.

**Review by shruthipatil11 — APPROVED** [review]

**Pull request merged**